airgun/entities/hostcollection.py

```diff
@@ -172,6 +176,33 @@ def install_errata(self, entity_name, errata_id,
             )
             return job_status_view.overview.read()
 
+    def manage_module_streams(self, entity_name, action_type,
+                              module_name, stream_version):
+        """Manage module streams (install, remove, update, reset)
+
+        :param str entity_name:  The host collection name.
+        :param action_type: remote action to execute. This is dict with containing 'Action'
+            and 'is_Customize' keys. Action key value can be one of 5:
+            'Enable', 'Disable', 'Install', 'Update', 'Remove', 'Reset'
```

Comment:
Same as in `entities/contenthost.py` - documentation is not consistent with code.

Reply:
done !